src/lib/plugins/index.ts

```diff
@@ -1,28 +1,28 @@
-module.exports.builtinPlugins = null
+export let builtinPlugins = null as any // todo
 
 const filterKeys = (keys) => {
   if (process.platform === 'browser') keys = keys.filter(k => k !== './external.js')
   keys = keys.filter(k => k !== './index.js' && k !== 'index.js')
   return keys
 }
 
-module.exports.initPlugins = () => {
+export const initPlugins = () => {
   if (process.platform === 'browser') {
     const isWebpack = !!require.context
     if (isWebpack) {
-      const pluginsMap = require.context('./', false, /^(?!.*(?:external.js$)).*\.js$/)
-      module.exports.builtinPlugins = filterKeys(pluginsMap.keys()).map(k => pluginsMap(k))
+      const pluginsMap = require.context('./', false, /^(?!.*(?:external.[jt]s$)).*\.js$/)
+      builtinPlugins = filterKeys(pluginsMap.keys()).map(k => pluginsMap(k))
     } else {
       // esbuild custom plugin
-      const files = require(/* webpackIgnore: true */ 'esbuild-import-glob(path:.,skipFiles:index.js,external.js)')
-      module.exports.builtinPlugins = Object.values(files)
+      const files = require(/* webpackIgnore: true */ 'esbuild-import-glob(path:.,skipFiles:index.js,external.js,index.ts,external.ts)')
```

Comment:
since we now have build step I think it would be better to remove these workarounds and instead create index.js with all plugins included. So we don't need to handle different platforms